web/template/partial/course/manage/create-lecture-form.gohtml

```diff
@@ -1,127 +1,216 @@
+                </span>
+                Video Files
+            </li>
+        </ol>
 
-        <label x-show="!formData.premiere && !formData.vodup && !loading">
-            <span class="text-sm text-5">Lecture Hall</span>
-            <select class="tl-select" id="lectureHallBulkSelect" x-model="formData.lectureHallId">
-                <option class="text-gray-300" value="0">Self streaming
-                </option>
-                {{range .LectureHalls}}
-                    <option value="{{.Model.ID}}" class="text-gray-300">
-                        {{.Name}}
-                    </option>
-                {{end}}
-            </select>
-        </label>
+        <div class="form-container-body">
+            <div x-show="currentTab === -1" x-data="{ progress: { COMB: 0, PRES: null, CAM: null } }" @voduploadprogress.window="(e) => (progress = e.detail)">
+                <h3>Your lecture is being created ...</h3>
+                <ul x-show="formData.vodup" class="mt-2">
+                    <li>Combined Video: <b x-text="progress.COMB === null ? 'skipped' : `${progress.COMB}%`"></b></li>
+                    <li>Presentation Video: <b x-text="progress.PRES === null ? 'skipped' : `${progress.PRES}%`"></b></li>
+                    <li>Camera Video: <b x-text="progress.CAM === null ? 'skipped' : `${progress.CAM}%`"></b></li>
+                </ul>
+            </div>
 
-        <label for="start" x-show="!loading">
-            <span class="text-sm text-5">Start</span>
-            <input class="tl-input" name="start" placeholder="2021-04-08 14:00" id="start" x-model="formData.start"
-                   x-ref="start"
-                   x-init="flatpickr($refs.start, {enableTime: true, time_24hr: true, altInput:true, altFormat:'Y-m-d H:i', dateFormat:'Z', allowInput: true})"
-                   @change="() => setTimeout(() => { regenerateRecurringDates(); recalculateDuration(); }, 100)"
-                   required/>
-        </label>
-        <template x-if="!formData.premiere && !formData.vodup && !loading">
-            <label for="end">
-                <span class="text-sm text-5">End</span>
-                <span x-show="formData.formatedDuration !== ''" class="text-sm font-light opacity-75"
-                      x-text="`( ${formData.formatedDuration} )`"></span>
-                <input class="tl-input" name="end" placeholder="16:00" id="end" x-model="formData.end" x-ref="end"
-                       x-init="flatpickr($refs.end, {enableTime: true, noCalendar: true, dateFormat: 'H:i', time_24hr: true, allowInput: true })"
-                       @change="() => setTimeout(() => recalculateDuration(), 100)"
-                       :required="!formData.vodup"/>
-            </label>
-        </template>
+            <div x-show="currentTab === 0" class="flex py-4">
+                <ul class="w-full">
+                    <li class="mb-2">
+                        <button
+                            @click="() => updateType(false)"
```

Comment:
IMO it would be nice if the page automatically switched to the second page without clicking continue here.